src/types/common/contactPoint.ts

```diff
@@ -36,19 +36,25 @@ export const contactPointSchema = () =>
     // Fax numbers (also using phone validation since they follow same format)
     z.object({
       system: z.literal("fax"),
-      value: validators().phoneNumber.required,
+      value: z.string().min(1, { message: t("field_required") }),
       use: z.enum(ContactPointUses),
     }),
     // Email addresses
     z.object({
       system: z.literal("email"),
-      value: z.string().email(),
+      value: z
+        .string()
+        .min(1, { message: t("field_required") })
+        .email(),
```

Comment:
wouldn't `required_error` option on string suffice? 🤔 

```suggestion
        .string({ required_error: t("field_required") })
        .email(),
```

Reply:
will update and test